File: loan/src/main/java/fr/iut/loan/business/LoanRequest.java
package fr.iut.loan.business;

import javax.validation.constraints.NotNull;

public class LoanRequest {
	
	@NotNull
	private Long accountId;
	
	private Integer amount;

	public Long getAccountId() {
		return accountId;
	}

	public void setAccountId(Long accountId) {
		this.accountId = accountId;
	}

	public Integer getAmount() {
		return amount;
	}

	public void setAmount(Integer amount) {
		this.amount = amount;
	}
}

File: client/src/AppBundle/Entity/LoanRequest.php
<?php


namespace AppBundle\Entity;

class LoanRequest
{
    private $accountId;

    private $amount;

    /**
     * @return mixed
     */
    public function getAccountId()
    {
        return $this->accountId;
    }

    /**
     * @param mixed $accountId
     */
    public function setAccountId($accountId)
    {
        $this->accountId = $accountId;
    }

    /**
     * @return mixed
     */
    public function getAmount()
    {
        return $this->amount;
    }

    /**
     * @param mixed $amount
     */
    public function setAmount($amount)
    {
        $this->amount = $amount;
    }

    public function getApiArray()
    {
        $request = array(
            "accountId" => $this->getAccountId(),
            "amount" => $this->getAmount()
        );

        return $request;
    }

    public function getApiWithoutAmountArray()
    {
        $request = array(
            "accountId" => $this->getAccountId()
        );

        return $request;
    }
}